foundation/plugins/org.jboss.tools.foundation.ui/src/org/jboss/tools/foundation/ui/credentials/internal/CredentialPreferencePage.java

```diff
@@ -365,11 +373,20 @@ private void addUserPressed() {
 		}
 
 	}
+
+	private static final String UNABLE_TO_DELETE_CREDENTIALS = "Unable to delete credentials stored in secure storage without authenticating to secure storage";
```

Comment:
Please put this in a proper externalized class...   Specifically  CredentialMessages.java and CredentialMessages.properties. 

Otherwise, great patch.